### ISSUE TITLE
change labels for "disease" and "disorder"

### ISSUE THREAD

**Issue by cmungall** [OGMS/ogms#176]
OGMS takes two strings that are part of non-ontological discourse that have complicated but often interchangeable usage and imposes ontological commitments on them. This leads to a lot of confusion.

Names are important. I recommend changing to "pathological organism disposition" and "pathological material entity" or something like that.

**Comment by zhengj2007** [comment]
The labels proposed by @cmungall look good to me. However, 'disease' and 'disorder' are commonly used terms in the biomedical areas. Do have any plan to add the 'disease' and 'disorder' as synonyms of "pathological organism disposition" and "pathological material entity"? not necessary exact synonym, might be narrow/broad/related synonym

**Comment by addiehl** [comment]
I think if we change the labels, we definitely need to retain the previous labels as some type of synonym.

**Comment by wdduncan** [comment]
I agree with @scheuerm that OGMS not having the terms 'disease' or 'disorder' is a bit odd. But, IMHO, the benefits outweigh the cost.  For example, see the [debate](https://github.com/OBOFoundry/COB/pull/226) in COB about this. We can (of course) add synonyms for whichever labels we choose. 

However, if OGMS is going to change its labels, MONDO and DO (and others) should change its labels in order to hopefully avoid confusion across Foundry.  I've submitted an [issue](https://github.com/monarch-initiative/mondo/issues/5998) for this on the MONDO tracker.

**Comment by sivaramarabandi** [comment]
I am not opposed to adding "pathological organism disposition" and
"pathological material entity"  as skos:altLabel to the terms 'Disease' and
'Disorder' if it will help clarify the meaning.

cheers
Sivaram
______________________
Sivaram Arabandi, MD, MS

ONTOPRO

W:  http://ontopro.com/
Ph: 832.726.2322
Li : https://www.linkedin.com/in/sivaramarabandi/

Think Semantics. Tame Silos.

**Comment by linikujp** [comment]
Hi all,

Interesting discussion! I've been asked about this question just recently,
and been posted about this as an example of how important it is to
harmonize ontology and provide a clean, non-redundant and coherent
ontologies to the end user who don't usually build ontologies.

My suggestion: Replace the label of term "disorder" to "pathological
material entity" in OGMS and make 'disease or disorder' an alternative term
with COB ID in OGMS.
Reason: The BFO:style "disorder" is a very BFO-level philosophical term,
which elucidates the full ontological definition of disease. Disorder (as a
'pathological material entity'), but it does cause confusion for end users
who don't know the BFO ontological definition of disease, because as Chris
M. said, end users use disorder and disease exchangeably. The creation of
BFO: disorder in deed causes user's confusion.

I think the advantages of my suggestion are: 1. keeps the use of disease
and disorder in OGMS, and 2. maintains a consistency with COB.
Before making changes of the "disorder" in OGMS, we might need to double
think of the use of the label "pathological material entity" to avoid the
confusion by its face value. Can we use 'pathologically disordered material
entity" instead?

Thanks,,
Asiyah

**Comment by alanruttenberg** [comment]
I support the change, with the old labels being added as alternative terms, and other labels as alternative terms, as desired.

**Comment by linikujp** [comment]
To summary, this is my suggestion:
1.  change "disorder" to "pathologically disordered material entity" and keep "disorder" as alternative terms with detailed explanation of the change.
2. keep "disease" and add "disease or disorder" as alternative term, and add explanation as well. 
Do you agree with above, Alan?

**Comment by alanruttenberg** [comment]
@linikujp I'm not thrilled with the "disease or disorder" as an alternative label, but wouldn't oppose it, push come to shove. The reason is that we don't generally recommend disjunctive classes in ontologies, and the label suggests that. But it's only a label and as @wdduncan  says, it's the definition that matters more.

I think there's objection to keeping disease as the primary label, so I'm more inclined to support @cmungall's suggestion for pathological organism disposition, or something similar, with disease as an alternative label. Disease as an alternative label for disorder also seems appropriate, given what we've been told about usage in the wild.

**Comment by linikujp** [comment]
"pathological organism disposition" will then become a term that is not going to be easily understand by non-BFO people, although I think it is maybe a broader term than disease, can a symptom become a "pathological organism disposition"?

**Comment by alanruttenberg** [comment]
Schulz uses "pathological disposition", which is shorter. Symptom is another of those words that's used in several senses. For example, the usage "symptom: runny nose" could mean the disposition to have a runny nose, or a type of realization of the disposition. It's also used in the disorder sense, as in "symptom: rash", which could also mean a tendency to develop rashes, i.e. part of disease course in which a rash disorder is created. Language is not the ontologist's friend in this domain.

**Comment by alanruttenberg** [comment]
@linikujp Also, regarding not being understood, as with all labels we need to be training ontology users to read the definition to understand the term. Applications should be designed to make the definition hard to avoid. Also, there's no suggestion that the term "disease course" be changed, and there ought to be an axiom on disease/pathological disposition: realized in only (part of some disease course). Maybe that would help.

**Comment by alanruttenberg** [comment]
Maybe I'm mistaken. There is the suggestion of "pathological process", though I don't think that means the same thing as disease course, as the latter is the "The totality of all processes through which a given disease instance is realized." whereas I think a pathological process would be a part of that. But maybe the axiom would be realized in only pathological process. But then "pathological process" is currently defined as "A bodily process that is clinically abnormal.", and such things are not only realizations of dispositions, but also processes associated with injuries. It looks like changing the labels won't be a surgical change but will require rewording of other definitions and possibly adding new classes.

**Comment by linikujp** [comment]
I agree that both human and machine need to understand the definitions. But only hope that trained ontology users to read the definition won't be a way for ontology to go far. I think the labels need to be somehow touching the reality and intuitive, not everyone is Aristotle. 
As you mentioned @alanruttenberg and @zhengj2007 also mentioned, if labels is being changed, the definitions and axioms need to be changed and make it work. 
Symptom and sign are difficult to deal with ... I think end users want to use "running nose" and relate it to things it is supposed to relate to. Perhaps we could put them all as "pathological disposition", and the disease ontology, symptom ontology can use the term as their upper term. COB can then set up some relations such as characteristics to relate a disease with symptoms.  Would that be a possible path?

**Comment by addiehl** [comment]
While I agree that changing 'disorder' to 'pathologically disordered material entity' or 'pathological material entity' is an improvement useful to disambiguate the label and add face value, I am not sure that changing 'disease' to 'pathological organism disposition' or 'pathological disposition' helps much, and might rather confuse users, who would see familiar disease names as subclasses. We could simply call it 'disease disposition' to retain the face value of the word disease.

**Comment by alanruttenberg** [comment]
@addiehl I like that idea.

**Comment by linikujp** [comment]
Support the "disease disposition" idea too!

**Comment by wdduncan** [comment]
Thanks for the suggest @addiehl! In the context of OGMS, I like the label `disease disposition` better than `pathological disposition` too.

**Comment by cmungall** [comment]
Remember it’s really important to be consistent with actual usage. Everyone
will assume you mean predisposition

**Comment by hoganwr** [comment]
Do we have a definition or elucidation of 'pathological'?

In the context of OGMS, 'clinically abnormal' was the phrase, not 'pathological'

From the paper:

'Disorder' =def. – "A causally relatively isolated combination of physical components that is (a) clinically abnormal and (b) maximal, in the sense that it is not a part of some larger such combination."

'Pathological Process' =def. – "A bodily process that is a manifestation of a disorder."

"When we say that some bodily feature of an organism is clinically abnormal, this signifies that it: (1) is not part of the life plan for an organism of the relevant type (unlike aging or pregnancy), (2) is causally linked to an elevated risk either of pain or other feelings of illness, or of death or dysfunction, and (3) is such that the elevated risk exceeds a certain threshold level"

Although it would smack of circularity, could add synomyms to disorder of "clinically abnormal body part" and "clinically abnormal anatomical entity".

**Comment by linikujp** [comment]
@cmungall  it is better to add disposition to disease, so that it is friendly and clear to the end user - who are unfamiliar with the BFO approach. Although we want them to read the definition, a lot of them still have their mental definition when they read labels.

Support @hoganwr's point on elucidation of "pathological".

**Comment by addiehl** [comment]
@cmungall Keep in mind that 'pathological material entity' does not match usage in the medical literature either as far as I know. We are in the process of making some compromises in regards to labels to use in OGMS and possibly other ontologies. I am not completely thrilled by 'disease disposition' in part for the reason you suggest, but the label does at least include the word disease and imply that it is a type of disposition, which is not wrong.

**Comment by cmungall** [comment]
The general principle is:

- if the concept corresponds to the same concept used by an ordinary
scientist or clinician, use the standard label
- if you are creating a novel abstraction, don't use a label that confuses
with an existing concept

**Comment by scheuerm** [comment]
@alanruttenberg I don't agree that "For example, the usage "symptom: runny nose" could mean the disposition to have a runny nose, or a type of realization of the disposition."  A runny nose is a realization.

**Comment by scheuerm** [comment]
It seems like we are approaching a consensus on this.  So the proposal would be to:

- Change OGMS:disorder => OGMS:pathologically disordered material entity, with disorder as a synonym and =def. – "A causally relatively isolated combination of physical components that is (a) clinically abnormal and (b) maximal, in the sense that it is not a part of some larger such combination."
- Change OGMS:disease => OGMS:disease disposition, with synonym disease (but not disorder) and =def. - "A disposition (i) to undergo pathological processes that (ii) exists in an organism because of one or more disorders in that organism."
- Add OGMS: disease predisposition with =def. something like "a disposition to form a pathologically disordered material entity" and perhaps "disorder predisposition" as a synonym.
- Define OGMS:pathological that is consistent with the clinically abnormal elucidation
